packages/frontend/src/components/message/MessageMetaData.tsx

```diff
@@ -53,7 +53,8 @@ export default function MessageMetaData(props: Props) {
       {/* FYI the email doesn't need `aria-live`
       as we don't expect it to get removed. See
       https://github.com/deltachat/deltachat-desktop/pull/5023#discussion_r2059382983 */}
-      {!encrypted && (
+      {!encrypted && status !== 'in_noticed' && (
+        // if a message is in status in_notice we don't know if it is encrypted or not
```

Comment:
this seems wrong. 

you can see this also when opening a classic mail chat with fresh messages, the last message initially does not have an envelope as it is not noticed yet (but fresh)

i think, what is meant is `downloadState != 'done'`